packages/headless/src/controllers/result-list/headless-result-list.ts

```diff
@@ -18,7 +18,7 @@ export type {ResultListOptions, ResultListProps, ResultListState, ResultList};
  * @returns A `ResultList` controller instance.
  */
 export function buildResultList(
-  engine: SearchEngine,
+  engine: CoreEngine,
```

Comment:
This is the function that builds the Headless Search Result controller, I don't think it should receive a CoreEngine. it's specific to search.

Reply:
you are right, must have forgotten this one as the insight has a InsightSearchEngine. Good catch